docs/02-app/01-building-your-application/08-deploying/01-production-checklist.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Going to Production
+description: Before taking your Next.js application to production, here are some recommendations to ensure the best user experience.
+---
+
+Before taking your Next.js application to production, here are some recommendations to ensure the best user experience.
+
+## Overview
+
+- Use [caching](/docs/app/building-your-application/caching) wherever possible.
+- Ensure your database and backend are deployed in the same region.
+- Aim to reduce client-side JavaScript bundle sizes by [moving the "use client" boundaries down the React component tree](https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#moving-client-components-down-the-tree).
```

Comment:
We don't have to recommend moving the use client boundaries down, instead it should explain leveraging Server Components. It's not exactly that the tree stops when there is a client component, they can wrap server components content.

Reply:
I think we're saying the same thing here but with different words. One mistake I often see people doing is opting into Client Components higher up the tree and/or passing large data objects to the client, thus increasing their JavaScript bundle sizes. 

Hence why it might be good to be explicit about "moving the boundaries down the tree" or in other words moving client components down the leaves, and linking to the docs section that recommends this pattern.

Since Server Components are the default, it's more about asking people to be mindful of when/where they place the client boundaries.

Reply:
I'll write a suggestion below 🙏🏼